### PR TITLE
Editorial cleanup

### LINKED ISSUE #7: add non-normative examples of API
I think it would be worthwhile to add at least one non-normative example of the UUID API in action.

The [kv-storage spec](https://wicg.github.io/kv-storage/#intro) has some great examples we could use for inspiration.

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #7 

Cleanups ReSpec usage and cross references. 

Makes the following editorial changes: 

* Moved motivation to README. 
* Added implementation status to README. 
* Added example to README.
* Added example from #7 to spec.
* Fixed up conformance wording for calling randomUUID()

***
<a href="https://pr-preview.s3.amazonaws.com/marcoscaceres/uuid/pull/28.html" title="Last updated on Aug 20, 2021, 5:04 AM UTC (63b54cc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/uuid/28/5f620e9...marcoscaceres:63b54cc.html" title="Last updated on Aug 20, 2021, 5:04 AM UTC (63b54cc)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,223 +1,226 @@
...
-        defined in [[!WebCryptoAPI]].
+        This is a work in progress.
       </p>
+    </section>
+    <section>
+      <h2>
+        Usage Example
+      </h2>
+      <p>
+        To generate a UUID, simply call {{Crypto}}'s {{Crypto/randomUUID()}}
```
Comment (bcoe) [inline index.html:59]: The linter is claiming that this is a _"normative reference to a non normative term"_.

**Review by bcoe — CHANGES_REQUESTED** [review]
Edits look great, 👍 thanks for the help.

Left one note about the linter flagging `{{Crypto}}`.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:59]: Yeah, ignore that. I’ll get that fixed upstream.

Comment (marcoscaceres) [reply index.html:59]: Filed https://github.com/w3c/webref/issues/329

**Review by bcoe — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1,223 +1,226 @@
...
+          The `randomUUID()` method
+        </h3>
         <p>
-          For the steps described in the [=generate a random UUID=] algorithm,
-          the <dfn class="dfn-paneled" data-dfn-type="dfn" data-export data-lt="hexadecimal representation" id="hex-representation">hexadecimal representation</dfn>
-          of a <a data-cite="INFRA#byte">byte</a> <var>value</var> is the
-          two-character string created by expressing <var>value</var> in hexadecimal
-          using <a data-cite="INFRA#ascii-lower-hex-digit">ASCII lower hex digits</a>,
-          left-padded with <code>"0"</code> to reach two characters.
+          When <dfn>randomUUID()</dfn> is called, a [=user agent=] method MUST
```
Comment (domenic) [inline index.html:91]: This change is very incorrect. Web IDL says you are supposed to use the previous form, and Infra says you are not supposed to capitalize MUST.

Please revert this to the previous wording.

**Comment by domenic** [comment]
I think there were a number of unfortunate editorial changes made here, in addition to the incorrect one mentioned in my review. E.g. the proliferation of sections, or the blurring of the line between spec and explainer.

I'd encourage a bit more review and discussion before making such changes in the future.

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html:91]: Oops, sorry. My mistake. That’s an easy fix.

**Comment by marcoscaceres** [comment]
> proliferation of sections

it’s not a “preference”. ReSpec needs them to work properly, @domenic.

**Comment by domenic** [comment]
That's really unfortunate. I'll be sure to more strongly recommend a switch to Bikeshed in the future.

**Comment by marcoscaceres** [comment]
It’s being merged into Web Crypto, which is ReSpec. Please stop complaining about little changes I made. The spec is was simpler in changes because of the changes I made.

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html:91]: https://github.com/WICG/uuid/pull/31

**Pull request merged**